src/pyfaf/cmdline.py

```diff
@@ -191,6 +191,16 @@ def add_repo_type(self, choices=None, helpstr=None):
         """
         self.add_argument("--type", choices=choices, help=helpstr)
 
+    def add_ext_instance(self, multiple=False, helpstr=None):
+        """
+        Add the `INSTANCE_ID` positional argument for an external FAF instance.
+        """
+        nargs = None
+        if multiple:
+            nargs = "*"
```

Comment:
What about `nargs = "*" if multiple else None` to make a bit shorter and Pythonic? (Also applicable elsewhere in this PR.)

Reply:
rewritten